### ISSUE TITLE
Unhandled Rejection (TypeError): Object(...) is not a function

### ISSUE THREAD

**Issue by dadaphl** [celo-org/react-celo#48]
when I try to connect a wallet, after clicking on any provioder (wallet connect, metamask ect.) when the QR code is being generated I get this screen and QR code is never generated

![Screenshot-20211014111727-1020x551](https://user-images.githubusercontent.com/415536/137289048-d2f0a359-4316-4fa8-a8cd-b5b67e3ae871.png)

I created a new React app with `npx create-react-app` yarn add the use-contractkit according to the documentation. Modiefied package.json according to #45 

```
➜  celo-react-01 git:(master) ✗ node -v
v14.18.0
➜  celo-react-01 git:(master) ✗ yarn -v
1.22.15
➜  celo-react-01 git:(master) ✗ npx -v
8.0.0
➜  celo-react-01 git:(master) ✗ cat package.json
{
  "name": "celo-react-01",
  "version": "0.1.0",
  "private": true,
  "dependencies": {
    "@celo-tools/use-contractkit": "^1.2.0",
    "@testing-library/jest-dom": "^5.11.4",
    "@testing-library/react": "^11.1.0",
    "@testing-library/user-event": "^12.1.10",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-scripts": "4.0.3",
    "web-vitals": "^1.0.1"
  },
  "resolutions": {
    "**/@walletconnect/utils": "2.0.0-beta.0"
  },
  "scripts": {
    "start": "react-scripts start",
    "build": "react-scripts build",
    "test": "react-scripts test",
    "eject": "react-scripts eject"
  },
  "eslintConfig": {
    "extends": [
      "react-app",
      "react-app/jest"
    ]
  },
  "browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
}
➜  celo-react-01 git:(master) ✗ uname -a
Linux rex 5.11.0-37-generic #41-Ubuntu SMP Mon Sep 20 16:39:20 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by tunjayhuseynov** [comment]
![C5DB3354-8948-4D50-8ACD-72DC40A60067](https://user-images.githubusercontent.com/37210254/137518086-012f81bf-8b33-431c-8f3b-c8f88daada9c.png)

I did it, it solved, but other issues started to happen

**Comment by aslawson** [comment]
We fixed #45 in the new version of use-contractkit. We will check back after it is released
